### PR TITLE
sys-cluster/glusterfs: version 10.2

### PR DESCRIPTION
**Pull request by jkroonza**
No major changes, mostly minor bug fixes, except for one SSE related
issue where certain structures needs to be aligned.

Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Jaco Kroon <jaco@uls.co.za>

### CODE REVIEW

File: sys-cluster/glusterfs/files/glusterd-10.2.initd [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+start_pre() {
+	# Ensure that the GlusterFS auxiliary mount parent directory exists
+	checkpath --directory --owner gluster:gluster --mode 0775 /run/gluster
+}
+
+start_post() {
+	local c=0
+	ebegin "Waiting for glusterd to start up"
+	while ! /usr/sbin/gluster volume list >/dev/null 2>&1 && [ "${c}" -lt "${glusterd_max_wait_start-60}" ]; do
+		$(( ++c ))
```
Comment (juippis) [inline sys-cluster/glusterfs/files/glusterd-10.2.initd:26]:
```
 * QA Notice: shell script appears to use non-POSIX feature(s):
 *    possible bashism in /etc/init.d/glusterd line 26 ('$((++n))' should be '$((n=n+1))'):
 *    		$(( ++c ))
```
You can install dev-util/checkbashisms to run these checks.

File: sys-cluster/glusterfs/files/glusterd-10.2.initd [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/sbin/openrc-run
+# Copyright 1999-2015 Gentoo Foundation
```
Comment (juippis) [inline sys-cluster/glusterfs/files/glusterd-10.2.initd:2]: Update the copyright year? :)

File: sys-cluster/glusterfs/files/glusterfsd-10.2.initd [hunk 3]
```diff
@@ -0,0 +1,121 @@
+#!/sbin/openrc-run
+# Copyright 1999-2012 Gentoo Foundation
```
Comment (juippis) [inline sys-cluster/glusterfs/files/glusterfsd-10.2.initd:2]: Here too.

File: sys-cluster/glusterfs/glusterfs-10.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,199 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
```
Comment (juippis) [inline sys-cluster/glusterfs/glusterfs-10.2.ebuild:6]: Drop 3.7 where ever possible.

File: sys-cluster/glusterfs/files/glusterd-10.2.initd (hunk 2 above)
Comment (jkroonza) [reply sys-cluster/glusterfs/files/glusterd-10.2.initd:2]: And of course Foundation to Authors?

File: sys-cluster/glusterfs/files/glusterd-10.2.initd (hunk 1 above)
Comment (jkroonza) [reply sys-cluster/glusterfs/files/glusterd-10.2.initd:26]: Thank you!!!!!!

File: sys-cluster/glusterfs/files/glusterd-10.2.initd (hunk 2 above)
Comment (juippis) [reply sys-cluster/glusterfs/files/glusterd-10.2.initd]: Heh, yeah.

**Pull request closed without merging**